.storybook/main.js

```diff
@@ -13,11 +13,21 @@ module.exports = {
     path.resolve(__dirname, 'theme-selector/register'),
   ],
   webpackFinal: async (config, { configType }) => {
+    const fileLoaderRule = config.module.rules.find(rule => rule.test.test('.svg'));
```

Comment:
This looks prone to breaking. Best off looking for the file loader itself. You should be able to filter by constructor name.